Language/Mixins/Mixin_Application/interfaces_t01.dart

```diff
@@ -2,26 +2,25 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion If the mixin application declares support for interfaces, the
-/// resulting class implements those interfaces.
-/// @description Checks that if the mixin application declares support for
-/// interfaces and the resulting class implements those interfaces then no
-/// static warning occurs
+/// @assertion Let D be a mixin application class declaration of the form
+/// abstract? class N = S with M1, ..., Mn implements I1, ..., Ik;
+/// ...
+// The effect of D in library L is to introduce the name N into the scope of L,
+// bound to the class defined by the clause S with M1, ..., Mn with name N,
```

Comment:
Should these two lines be a doc-comment starting with `///` as the comments above and below it?